cli/alpine-basic/README.md

```diff
@@ -0,0 +1,20 @@
+# Zowe CLI Docker Container
+
+This repository contains the files required to create a Zowe CLI docker container with root access on Alpine
+Does not install the DB2 plug-in
```

Comment:
Should we mention the reason why? I assume because it's not supported on some architectures such as ARM 😋